Resources/Prototypes/_Harmony/Entities/Objects/Misc/identification_cards.yml

```diff
@@ -0,0 +1,13 @@
+﻿# Harmony base ID card
```

Comment:
It's not a base card. Either remove the comment, or state that this is a Poquito ID card